packages/flame_bloc/test/flame_bloc_game_test.dart

```diff
@@ -96,6 +96,7 @@ void main() {
         game.add(component);
 
         expect(component.state, equals(InventoryState.sword));
+        game.remove(component);
```

Comment:
Why was this line added?

Reply:
In order to get 100% coverage for flame_bloc repository.

Reply:
But it is kind of cheating, this test doesn't test anything regarding removal?

Reply:
Fair, i've expanded this to a bigger test